CHANGELOG.md

```diff
@@ -1 +1,2 @@
 2015-11-07 - Initial release
+2015-12-17 - Adding authentication
```

Comment:
I know this isnt todays date but I thought the actual date of when the functionality was added would be more appropriate